lib/ext/heroku/command/addons.rb

```diff
@@ -203,7 +203,14 @@ def create
 
       action("Creating #{addon['name'].downcase}") {}
       action("Adding #{addon['name'].downcase} to #{app}") {}
-      action("Setting #{addon['config_vars'].join(', ')} and restarting #{app}") do
+
+      if !addon['config_vars'].empty?
+        message ="Setting #{addon['config_vars'].join(', ')} and restarting #{app}"
+      else
+        message = "Restarting #{app}"
```

Comment:
Fixed in 1c3c67ee041f53988142b916ad13c0cef1e9b065 -- hopefully the `unless` to get rid of the negation isn't too controversial there! :wink: I felt like since it was a single branch conditional, an `unless` was ok.